content/blog/2021-04-19-experiment-refs.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Utilizing Custom Git References in DVC
+date: 2021-04-19
+description: |
+  In DVC 2.0, we’ve introduced a new feature set aimed at simplifying the
+  versioning of lightweight ML experiments. In this post, we’ll dive into
+  how exactly these new experiments work.
+
+descriptionLong: |
+  In DVC 2.0, we’ve introduced a new feature set aimed at simplifying the
```

Comment:
💅  it supports links and regular Markdown text styles. Should we link to the previous blog post, the release 2.0?